3_javascript/javascript3/week2/scripts.js

```diff
@@ -0,0 +1,85 @@
+let fromCurrency = 'EUR';
+let toCurrency = 'DKK';
+let amount = 0;
+
+document.addEventListener('DOMContentLoaded', () => {
+  const inputAmount = document.getElementById('amount');
+  const fromCurrencySelect = document.getElementById('fromCurrency');
+  const toCurrencySelect = document.getElementById('toCurrency');
+  const clickConvert = document.getElementById('convert');
+
+  function populateCurrencyDropdowns() {
+    const currencyPairs = [
+      'USD',
+      'EUR',
+      'DKK',
+      'JPY',
+      'GBP',
+      'AUD',
+      'CAD',
+      'CHF',
+      'CNY',
+      'SEK',
+      'NZD',
+      'SGD',
+      'HKD',
+      'NOK',
+      'KRW',
+      'MXN',
+      'INR',
+      'BRL',
+      'ZAR',
+      'AED',
+      'RUB',
+    ];
+
+    currencyPairs.forEach((currency) => {
+      const option = document.createElement('option');
+      option.value = currency;
+      option.text = currency;
+      fromCurrencySelect.appendChild(option);
+      toCurrencySelect.appendChild(option.cloneNode(true));
+    });
+
+    fromCurrencySelect.value = fromCurrency;
```

Comment:
You already have the default values for the ```fromCurrency``` and ```toCurrency``` set directly in the HTML. So you can skip this in the Javascript code.